CMakeLists.txt

```diff
@@ -61,7 +61,21 @@ endif()
 # Find OGRE
 list(APPEND ign_ogre_components "RTShaderSystem" "Terrain" "Overlay")
 
-ign_find_package(IgnOGRE VERSION 1.8.0
+# Ogre versions greater than 1.9 are not officialy supported.
+# Display a warning for the users on this setup unless they provide
+# USE_UNOFFICAL_OGRE_VERSIONS flag
+if (NOT USE_UNOFFICAL_OGRE_VERSIONS)
+  ign_find_package(IgnOGRE VERSION 1.10
+    COMPONENTS ${ign_ogre_components})
+
+  if (OGRE_FOUND)
+    IGN_BUILD_WARNING("Ogre 1.x versions greater than 1.9 are not officially supported."
+                      "The software might compile and event work but support from upstream"
```

Comment:
`event` - > `even`

> support from upstream could be reduced to accept patches for newer versions

To confirm, I think this means we will provided limited support in the form of accepting patches for newer versions but not actively develop against them right?

Reply:
> To confirm, I think this means we will provided limited support in the form of accepting patches for newer versions but not actively develop against them right?

Eh yes, that is my idea but you Ian or @chapulina probably have it clearer in your mind. Please feel free to reformulate to somehow explain that we won't fix bug on other versions of Ogre but we accept all kind of patches to make it happen.

Reply:
Got it. A couple of minor suggestions are: `event` -> `even`, and `accept` -> `accepting`

Reply:
Done  c57a8e5